inst/private/python_ipc_driver.m

```diff
@@ -33,10 +33,10 @@
 
   %% version check
   if exist('OCTAVE_VERSION', 'builtin')
-    if (compare_versions (OCTAVE_VERSION (), '4.2', '<'))
+    if (compare_versions (OCTAVE_VERSION (), '5.1', '<'))
       fprintf(['\n********************************************************************\n' ...
                'Your Octave version is %s but Octsympy is currently untested on\n' ...
-               'anything before 4.2.\n\n'], OCTAVE_VERSION ())
+               'anything before 5.1.\n\n'], OCTAVE_VERSION ())
```

Comment:
this feels right to me, but less certain about actively removing workarounds.

Reply:
I think we can also safely remove workarounds in ` %!test`.